### DISCUSSION TITLE
More types of favourites?

### DISCUSSION THREAD

**Discussion by hakonalstadheim** [mastodon/mastodon#24609] [General]
Often a post makes me want to send more than a star to the author, but the original toot stands best on its own, so a reply would be out of place. Just now, I'd have liked to send someone a big grin rather than a star. Is this something that has been considered? I realize this would probably also make _dis_likes the next logical request, but that is still a separate discussion

**Comment by shaedrich** [comment]
Emoji reactions – that would be nice
